### PR TITLE
[bitnami/nginx] stream server blocks

### PR DESCRIPTION
**Pull request by TrueGameover**
### Description of the change

Allow autoload stream servers configs from /opt/bitnami/nginx/conf/stream_server_blocks as /opt/bitnami/nginx/conf/server_blocks

### Benefits

Conveniently load stream configurations. It's not available now.

### Possible drawbacks

no

### CODE REVIEW

**Review by juan131 — COMMENTED** [review]
Thanks so much for this PR! Please check my comments

File: bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/nginx/conf/nginx.conf [hunk 1]
```diff
@@ -58,3 +58,7 @@ http {
         }
     }
 }
+
+stream {
+    include  "/opt/bitnami/nginx/conf/stream_server_blocks/*.conf";
+}
```
[the remark this answers is not in the document]
Comment (TrueGameover) [reply bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/nginx/conf/nginx.conf]: ok, I added it

File: bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/scripts/libnginx.sh [hunk 2]
```diff
@@ -208,6 +208,10 @@ nginx_initialize() {
     fi
     nginx_configure "absolute_redirect" "$(is_boolean_yes "$NGINX_ENABLE_ABSOLUTE_REDIRECT" && echo "on" || echo "off" )"
     nginx_configure "port_in_redirect" "$(is_boolean_yes "$NGINX_ENABLE_PORT_IN_REDIRECT" && echo "on" || echo "off" )"
+
+    if [[ "${NGINX_ENABLE_STREAM}" != "no" ]]; then
```
Comment (juan131) [inline bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/scripts/libnginx.sh:212]: You can use this instead:
```suggestion
    if is_boolean_yes "$NGINX_ENABLE_STREAM"; then
```

File: bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/nginx/conf/nginx.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (juan131) [reply bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/nginx/conf/nginx.conf]: thanks!

File: bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/scripts/libnginx.sh (hunk 2 above)
Comment (TrueGameover) [reply bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/scripts/libnginx.sh]: ok, I changed it

**Review by juan131 — APPROVED** [review]
Thanks so much! LGTM

**Comment by juan131** [comment]
@TrueGameover we just released new image versions (`1.26.2-debian-12-r6` and `1.27.1-debian-12-r5`) including your changes, thanks so much for the contribution.

**Pull request merged**